lib/adhearsion/script_ahn_loader.rb

```diff
@@ -8,9 +8,22 @@ module ScriptAhnLoader
     RUBY = File.join(*RbConfig::CONFIG.values_at("bindir", "ruby_install_name")) + RbConfig::CONFIG["EXEEXT"]
     SCRIPT_AHN = File.join('script', 'ahn')
 
+    def self.load_script_ahn(path = Dir.pwd)
+      path = Pathname.new(path).expand_path
+      begin
+        script = File.join(path, SCRIPT_AHN)
```

Comment:
mostly cause of explicitly checking whether the script path exists before trying to load it ... seemed "less confusing" than to rely on an implementation detail of `in_ahn_application?` 

```ruby
    def self.load_script_ahn(path = Dir.pwd)
      path = Pathname.new(path).expand_path
      begin
        if in_ahn_application?(path)
          load File.join(path, SCRIPT_AHN)
          return true
        end
        path = path.parent
      end until path.root?
      nil
    end
```

... but I do not care that much, if this the preferred way for you?